workshop/exercises/pygeoapi.config.yml

```diff
@@ -184,6 +184,37 @@ resources:
               data: tests/data/ne_110m_admin_0_countries.sqlite
               id_field: ogc_fid
               table: ne_110m_admin_0_countries
+    airports:
+        type: collection
+        title: airports of the world
+        description: airports of the world
+        keywords:
+            - airports
+            - natural earth
+        links:
+            - type: text/html
+              rel: canonical
+              title: information
+              href: https://www.naturalearthdata.com/
```

Comment:
Please provide the link to some place where we can find information about the airport file (e.g.: metadata). This is not the naturalearth dataset.

Reply:
You can put that exact link!